app/apptesting/test_suite.go

```diff
@@ -109,7 +116,8 @@ func (s *KeeperTestHelper) setupGeneral() {
 	}
 
 	s.SetEpochStartTime()
-	s.TestAccs = CreateRandomAccounts(3)
+	s.TestAccs = []sdk.AccAddress{}
+	s.TestAccs = append(s.TestAccs, baseTestAccts...)
```

Comment:
The failing go test seems to be related to this change

Reply:
ty! The test was accidentally depending on this being different in each core loop iteration